tests/test_optimizely_config.py

```diff
@@ -623,9 +777,51 @@ def setUp(self):
                             },
                             'id': '42224',
                             'key': 'group_2_exp_3',
-                            'audiences': ''
+                            'audiences': 'OR "Test attribute users 3"'
                         }
                     },
+                    'delivery_rules': [],
+                    'experiment_rules': [
+                        {
+                            'id': '42222',
+                            'key': 'group_2_exp_1',
+                            'variations_map': {
+                                'var_1': {
+                                    'id': '38901',
+                                    'key': 'var_1',
+                                    'feature_enabled': None,
+                                    'variables_map': {}
+                                }
+                            },
+                            'audiences': 'OR "Test attribute users 3"'
```

Comment:
```suggestion
                            'audiences': '"Test attribute users 3"'
```
When we have a single audience Id, clean up the prepending conditions. Same for other tests as well.

Reply:
corrected and tested